src/Sylius/Behat/Context/Setup/ProductContext.php

```diff
@@ -98,5 +98,6 @@ public function storeHasAProductPricedAt($productName, $price)
     public function productBelongsToTaxCategory(ProductInterface $product, TaxCategoryInterface $taxCategory)
     {
         $product->setTaxCategory($taxCategory);
+        $this->productRepository->add($product);
```

Comment:
Actually, we should use a manager here and call flush(). OR ideally flush once after whole background but we can't... :S